dist/obs-server.spec

```diff
@@ -664,6 +709,29 @@ usermod -a -G docker obsservicerun
 %ghost /srv/www/obs/api/log/production.log
 %ghost %attr(0640,root,www) %secret_key_file
 
+%{_unitdir}/obs-api-support.target
+%{_unitdir}/obs-clockworkd.service
+%{_unitdir}/obs-delayedjob-queue-consistency_check.service
+%{_unitdir}/obs-delayedjob-queue-default.service
+%{_unitdir}/obs-delayedjob-queue-issuetracking.service
+%{_unitdir}/obs-delayedjob-queue-mailers.service
+%{_unitdir}/obs-delayedjob-queue-project_log_rotate.service
+%{_unitdir}/obs-delayedjob-queue-quick.service
+%{_unitdir}/obs-delayedjob-queue-quick@.service
+%{_unitdir}/obs-delayedjob-queue-releasetracking.service
+%{_unitdir}/obs-sphinx.service
+/usr/sbin/rcobs-api-support
+/usr/sbin/rcobs-clockworkd
+/usr/sbin/rcobs-delayedjob-queue-consistency_check
+/usr/sbin/rcobs-delayedjob-queue-default
+/usr/sbin/rcobs-delayedjob-queue-issuetracking
+/usr/sbin/rcobs-delayedjob-queue-mailers
+/usr/sbin/rcobs-delayedjob-queue-project_log_rotate
+/usr/sbin/rcobs-delayedjob-queue-quick
+/usr/sbin/rcobs-delayedjob-queue-quick@
```

Comment:
I'm not sure this works. Have you tried?

Reply:
doesn't make much sense anyway - as rcX will forward to the X service, which does not exist (it's just a generator)

Reply:
`/usr/sbin/rcobs-delayedjob-queue-quick@` line removed.